play/dbt-materialize/README.md

```diff
@@ -0,0 +1,62 @@
+## dbt-materialize
+
+[dbt](https://www.getdbt.com/) adapter for [Materialize](http://materialize.io). 
+
+Note, this plugin is a work in progress, and not yet suitable for production.
+
+### Installation
+This plugin can be installed via pip:
+```
+$ pip install dbt-materialize
+```
+
+### Configuring your profile
+
+[Materialize](http://materialize.io) is based on the Postgres database protocols, so use the
+[dbt postgres settings](https://docs.getdbt.com/docs/profile-postgres) in your connection profile,
+only substitute `type: materialize` for `type: postgres`.
+
+## Supported Features
+
+### Materializations
```

Comment:
Are **Materializations** standard dbt config features? like, is that name clear to people in a way that they'll be able to use it?

Reply:
Yes! `Materializations` is the word that `dbt` uses to specify how a model is persisted: https://docs.getdbt.com/docs/building-a-dbt-project/building-models/materializations/